libraries/r-tensorflow-1.12/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# install miniconda, and set the appropriate path variables.
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.4.10-Linux-x86_64.sh -O ~/miniconda.sh && \
```

Comment:
Can this use https://repo.anaconda.com/miniconda/Miniconda3-4.6.14-Linux-x86_64.sh so we can just use the anaconda35 library and not duplicate this?

Reply:
there is actually something different around the installation here, I might backport it into anaconda proper. It looks like I can install and run everything as root and get things to work in algorithms

Reply:
but that might only be because they are installed in the dockerfiles, I have a feeling they would run into issues when being installed by the user with algorithm-build